Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:8.16-alpine as builder
```

Comment:
@SleeplessByte Should we use Node LTS here? To avoid having to change this hard-coded version frequently.

Reply:
Yes, Node LTS please!

Reply:
Changed the base image to lts-alpine